Models/ProjectModels/File/Manager.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+
+namespace Models\ProjectModels\File;
+
+use Interfaces\IDataManagement;
+
+/**
+ * @package Models\ProjectModels\File
+ */
+class Manager implements IDataManagement
+{
+    private const MEDIA_IMAGES_PATH = 'Media/images/';
+    private const USERS_IMAGES = 'users';
+    private const ADMIN_USER_IMAGES = 'admins';
+    private const BOOKS_LOGOS_IMAGES = 'books_catalog';
+    private const USERS_IMG_SIZE = 'users';
+    private const ADMINS_IMG_SIZE = 'admins';
+    private const BOOKS_IMG_SIZE = 'book';
+    private const IMG_MAX_SIZE = [
+        self::USERS_IMG_SIZE => 524288,
+        self::ADMINS_IMG_SIZE => 524288,
+        self::BOOKS_IMG_SIZE => 1048576
+    ];
+    private const IMAGE_FOLDER = [
+        self::USERS_IMAGES => 'users/',
+        self::ADMIN_USER_IMAGES => 'admin_users/',
+        self::BOOKS_LOGOS_IMAGES => 'books_logos/'
+    ];
+
+    private array $data;
+
+    public function __construct()
+    {
+        $this->data = $_FILES;
```

Comment:
You can check in the constructor is $_FILES array exist and if it's not throw an exception with a text like '$_FILES array is empty'